app/bundles/CoreBundle/Views/Standard/list.html.php

```diff
@@ -1,205 +0,0 @@
-<?php
-
-/*
- * @copyright   2014 Mautic Contributors. All rights reserved
- * @author      Mautic
- *
- * @link        http://mautic.org
- *
- * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
- */
-
-if ('index' == $tmpl) {
```

Comment:
Although removing this file without its twig counterpart will result with the same error mentioned in https://github.com/mautic/mautic/pull/12021

Reply:
> We'll need the standard list.html.twig. Ensure that the Marketing Messages UI is working.

That file already exists. I haven't seen any issues on the Marketing Messages UI.

Reply:
I missed that. Thanks for checking!